src/linux/backend-v4l2.cpp

```diff
@@ -655,7 +656,7 @@ namespace librealsense
                         // /sys/devices/pci0000:00/0000:00:xx.0/ABC/M-N/version
                         usb_specification = get_usb_connection_type(real_path + "/../../../");
                     }
-                    else // Video4Linux Devices that are not listed as UVC
+                    else // COMMENT THIS TO MAKE USB ENUMERATION POSSIBLE - Video4Linux Devices that are not listed as UVC
```

Comment:
What does this comment means?
Currently USB enumeration is not possible?

Reply:
This comment has been deleted in the dynamic enumeration PR - I will delete it here also